doc/classes/String.xml

```diff
@@ -670,8 +670,9 @@
 			</argument>
 			<description>
 				Splits the string by a [code]divisor[/code] string and returns an array of the substrings, starting from right.
-				[b]Example:[/b] [code]"One,Two,Three"[/code] will return [code]["One","Two","Three"][/code] if split by [code]","[/code].
-				If [code]maxsplit[/code] is specified, then it is number of splits to do, default is 0 which splits all the items.
+				[b]Example:[/b] [code]"One,Two,Three,Four"[/code] will return [code]["Three","four"][/code] if split by [code]","[/code] with maxsplit of [code]"2"[/code].
+				If [code]maxsplit[/code] is specified, then it is number of splits to do from the right up to maxsplits, default is 0 which splits all the items.
+				The return value will be ordered from the left ( 0 if default or maxsplits if set ) to the right ( the number of elements in the array ).
```

Comment:
The parentheses are not really clear (and there should not be spaces after `(` and before `)`. I'd suggest something like this instead:
```
The splits in the returned array are sorted in the same order as the original string, from left to right.
```
The example coming afterwards should be sufficient to make this clear.